tests/rbd/rbd_trash.py

```diff
@@ -0,0 +1,64 @@
+from tests.rbd.exceptions import ImageNotFoundError, RbdBaseException
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verify the trash functionality. Delete the images and check if they are moved to trash automatically
+
+    This module verifies trash operations
+
+    Args:
+        kw: test data
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+
+    Test case covered -
+    CEPH-83573297 - Enable "rbd_move_to_trash_on_remove" config setting, delete images
+    and check if they are moved to trash automatically.
+
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+       (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+       conf and keyring files
+
+    Test Case Flow:
+    1. Create a pool and an Image
+    2. generate IO in images
+    3. Enable "rbd_move_to_trash_on_remove" in client config
+    4. Delete images and verify the deleted images are moved to trash automatically
+
+    """
+    log.info("Running Trash function")
+    log.info("executing *****")
```

Comment:
Can we please merge these lines and have single info at the start of the module ?